### PR TITLE
test: enable --cgroup-parent test

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
Test still seems angry, but LGTM if you can convince it to pass

File: libpod/runtime_ctr.go [hunk 1]
```diff
@@ -343,9 +343,6 @@ func (r *Runtime) setupContainer(ctx context.Context, ctr *Container) (_ *Contai
 					if err != nil {
 						return nil, errors.Wrapf(err, "error retrieving pod %s cgroup", pod.ID())
 					}
-					if podCgroup == "" {
```
Comment (cdoern) [inline libpod/runtime_ctr.go:346]: possibly a warn here so user knows pod cgroup is not set? is this handled elsewhere?

**Review by cdoern — COMMENTED** [review]
If I recall correctly, I modified some of the cgroup/cgroup parent code when redesigning infra. There seemed to be some miscommunication on the default assignment of the cgroup path when rootless/rootful. That could be what is giving us some trouble here.

File: libpod/runtime_ctr.go (hunk 1 above)
Comment (giuseppe) [reply libpod/runtime_ctr.go]: my mistake, this change doesn't belong to this PR.  I've dropped it.

Thanks for noticing it!

**Comment by giuseppe** [comment]
tests are finally green

**Comment by mheon** [comment]
LGTM

**Pull request merged**